### PR TITLE
Extend single block FakeLevel

### PR DESCRIPTION
**Pull request by Nightenom**
# Changes proposed in this pull request
- prepare for itemhandler fixes
- at Ray: merge at your own will

## Testing
- [x] Yes I tested this before submitting it.
- [ ] I also did a multiplayer test.

Review please

### CODE REVIEW

File: src/main/java/com/ldtteam/common/fakelevel/FakeLevel.java [hunk 1]
```diff
@@ -293,9 +293,15 @@ public BlockState getBlockState(final BlockPos pos)
         return levelSource.isPosInside(pos) ? levelSource.getBlockState(pos) : Blocks.AIR.defaultBlockState();
     }
 
+    int lastX, lastZ;
+    ChunkAccess lastChunk = null;
```
Comment (Raycoms) [inline src/main/java/com/ldtteam/common/fakelevel/FakeLevel.java]: I know it's just for this method, but we should really have this at the top

File: src/main/java/com/ldtteam/common/fakelevel/FakeChunk.java [hunk 2]
```diff
@@ -217,9 +222,15 @@ public LevelChunkSection[] getSections()
         return new LevelChunkSection[0];
     }
 
+    int lastY;
+    LevelChunkSection lastSection = null;
```
Comment (Raycoms) [inline src/main/java/com/ldtteam/common/fakelevel/FakeChunk.java]: same for this

File: src/main/java/com/ldtteam/blockui/BOGuiGraphics.java [hunk 3]
```diff
@@ -184,4 +185,9 @@ public void popMvPose()
         RenderSystem.getModelViewStack().popMatrix();
         RenderSystem.applyModelViewMatrix();
     }
+
+    public static double getAltSpeedFactor()
```
Comment (Raycoms) [inline src/main/java/com/ldtteam/blockui/BOGuiGraphics.java:189]: What is this?

Comment (Nightenom) [reply src/main/java/com/ldtteam/blockui/BOGuiGraphics.java:189]: Temporary thing to allow fast zoom/drag/scroll when holding alt, cuz i have apparently high dpi monitor now and it was super slow

Comment (Raycoms) [reply src/main/java/com/ldtteam/blockui/BOGuiGraphics.java:189]: xD

**Pull request merged**